src/components/Button.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
-Button.propTypes = {
+styled.button.propTypes = {
```

Comment:
Didn't know this works! I would have just deleted the propTypes as a styled-component doesn't have restrictions on what props you can use – as long as they are allowed on HTMLElements.